### PR TITLE
Make tab(\t) a valid whitespace character.

### PR DESCRIPTION
**Pull request by Alek900**
Some prefer to use tab for indentation.

### CODE REVIEW

File: src/wren_compiler.c [hunk 1]
```diff
@@ -789,6 +789,7 @@ static void nextToken(Parser* parser)
         makeToken(parser, TOKEN_LINE);
         return;
 
+      case '\t':
       case ' ':
         // Skip forward until we run out of whitespace.
         while (peekChar(parser) == ' ') nextChar(parser);
```
Comment (ryanplusplus) [inline src/wren_compiler.c]: I think this should be:

    while (peekChar(parser) == ' ' || peekChar(parser) == '\t') nextChar(parser);

**Comment by ryanplusplus** [comment]
Where are your tests?

**Comment by munificent** [comment]
People who use tabs are heathens! OK, not really. I like this change. :smile: 

Can you add some tests? You can just add new test files to the root of the `test` directory. Mainly I just want to see that it (1) handles tabs are a separator between tokens, and (2) can handle skipping over a mixture of tabs and spaces (which is the issue @ryanplusplus methods above).

Thanks!

**Comment by munificent** [comment]
Sorry this fell through the cracks for a while. It's merged now. Thank you!

If you'd like, you're more than welcome to send me a pull request to add yourself to the AUTHORS file. :grinning:

**Pull request merged**